app/javascript/utils/strings.js

```diff
@@ -13,3 +15,50 @@ export const getIndicator = (forestType, landCategory) => {
   }
   return 'admin';
 };
+
+// formats/create ellipsis nodes text using # of lines available, node width, etc
+export const splitSVGText = (
+  text,
+  height,
+  _labelCharHeight,
```

Comment:
this nomenclature does seem very reusable friendly? Should it be a global util?

Reply:
I agree, but it has to do with text. I thought this could be a good place, but maybe some other `utils` file would be more appropriate?